### ISSUE TITLE
ETABS_Toolkit: Spaces in Properties names

### ISSUE THREAD

**Issue by kThorsager** [BHoM/ETABS_Toolkit#202]
#### Description:
While trying to push to ETABS I noticed that if a Panels ConstantThickness Property's Name (probably the others too) contains a space then the push component does nothing and it's success = False
(no name caused the same issue)

#### How to replicate:
Try to push a Panel with ConstantThickness where the ConstantThicness has a name with a space or no name

#### Expected behaviour:
something like replacing all " " with "-" with an accompanying warning about the change
perhaps produce standard names for those without or inform the user of the issue

**Comment by JosefTaylor** [comment]
Good catch. ETABS allows spaces, so I think this one is a BHoM screwup. I'll see if I can replicate.

**Comment by kThorsager** [comment]
Sorry tried to edit. the spaces were not a problem, I was just a bit quick on creating the issue and later tried to edit that out. The no name situation exists however

**Comment by kThorsager** [comment]
Another issue was however that `push` did not inform the user that the panels had no thickness and instead only displayed success = False. Similarly it would be helpful with an error message explaining what went wrong. 
Should I create an separate issue for that?

**Comment by JosefTaylor** [comment]
That is covered by issue #130 - I can probably take care of both at once.

**Comment by JosefTaylor** [comment]
@kThorsager if you don't mind testing the PR that's open, I believe I've fixed your issue.
